docs/src/appInsights.ts

```diff
@@ -0,0 +1,21 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ReactPlugin } from "@microsoft/applicationinsights-react-js";
+import { ApplicationInsights } from "@microsoft/applicationinsights-web";
+const reactPlugin = new ReactPlugin();
+
+const appInsights = new ApplicationInsights({
+	config: {
+		connectionString:
+			"InstrumentationKey=8bce9107-e3f7-445c-be87-153f7ecbbe47;IngestionEndpoint=https://centralus-2.in.applicationinsights.azure.com/;LiveEndpoint=https://centralus.livediagnostics.monitor.azure.com/;ApplicationId=11a3a32b-28c1-4f28-97fb-4b113a61c1e9",
+		enableAutoRouteTracking: true,
+		enableDebug: true,
+		extensions: [reactPlugin],
```

Comment:
Using the react plugin here looks to give you component level tracking but it looks like you need to use a higher order component for it to work, for example `export default withAITracking(reactPlugin, MyComponent);` (see https://www.npmjs.com/package/@microsoft/applicationinsights-react-js) 

I'm not sure if enabling this plugin does anything without the use of this higher order component?